ray-operator/controllers/common/pod.go

```diff
@@ -213,8 +187,8 @@ func setContainerEnvVars(container *v1.Container, rayNodeType rayiov1alpha1.RayN
 	if container.Env == nil || len(container.Env) == 0 {
 		container.Env = []v1.EnvVar{}
 	}
-	if !envVarExists("RAY_IP", container.Env) {
-		ip := v1.EnvVar{Name: "RAY_IP"}
+	if !envVarExists(RayIpEnv, container.Env) {
```

Comment:
for env vars, the convention is upper case like: RAY_IP
we do not need the `env` since it is known

Reply:
Sounds good. I will update it